src/Form/index.tsx

```diff
@@ -577,27 +583,9 @@ function Form({
       integrations.stytch?.metadata ?? integrations.firebase?.metadata;
     const authSteps = metadata?.auth_gate_steps ?? [];
     if (authSteps.length > 0) {
```

Comment:
this authSteps check can probably also be in getNextAuthStep, and just return '' if there is no next authStep